Microsoft.HealthVault.Fhir/Transformers/ObservationToHeight.cs

```diff
@@ -18,7 +19,10 @@ internal static Height ToHeight(this Observation observation)
             var height = observation.ToThingBase<ItemTypes.Height>();
 
             height.Value = ObservationToHealthVault.GetThingValueFromQuantity<Length>(observation.Value as Quantity);
-            height.When = ObservationToHealthVault.GetHealthVaultTimeFromEffectiveDate(observation.Effective);
+
+            if (observation.Effective == null)
```

Comment:
Close/open brackets. Also, if we are throwing an exception a more descriptive exception is better: "Effective field with a valid Effectiva date on observation is required". Although I am not sure why we should throw an exception here, what made you modify this?

Reply:
`Weight` and other `Observation` based types requires a `when` property